app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    with open(command.split()[1] + ".txt", "r") as text_file,\
```

Comment:
Save split result to a separate variable

Validate that the command length is equal exactly to 3 arguments

Validate that command name is exactly "cp"

Use unpacking to access command parts

Reply:
Done.